ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9306936?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9306936?buildTab=artifacts#/) on release-23.1 @ [a34af06d8e7230e4052db80d55da21a06013f81b](https://github.com/cockroachdb/cockroach/commits/a34af06d8e7230e4052db80d55da21a06013f81b):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1924075900
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1924075900
--- FAIL: TestTenantStreamingMultipleNodes (312.13s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
/cc @cockroachdb/disaster-recovery
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-26123

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9330870?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9330870?buildTab=artifacts#/) on release-23.1 @ [aec78f33d45a8376a0ecec885688bae60dbfb85c](https://github.com/cockroachdb/cockroach/commits/aec78f33d45a8376a0ecec885688bae60dbfb85c):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3770019881
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3770019881
--- FAIL: TestTenantStreamingMultipleNodes (287.89s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9354465?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9354465?buildTab=artifacts#/) on release-23.1 @ [7e72aae900c3ff4b44f1643c2d7ba55fbb2cbe23](https://github.com/cockroachdb/cockroach/commits/7e72aae900c3ff4b44f1643c2d7ba55fbb2cbe23):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes655755448
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes655755448
--- FAIL: TestTenantStreamingMultipleNodes (304.95s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9374725?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9374725?buildTab=artifacts#/) on release-23.1 @ [5d5abe704b0551556c35d51aa673d3162f493aa8](https://github.com/cockroachdb/cockroach/commits/5d5abe704b0551556c35d51aa673d3162f493aa8):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2582884448
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2582884448
--- FAIL: TestTenantStreamingMultipleNodes (309.66s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9384201?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9384201?buildTab=artifacts#/) on release-23.1 @ [63b683eab3e0ebfff812bb54fdac936c101d9383](https://github.com/cockroachdb/cockroach/commits/63b683eab3e0ebfff812bb54fdac936c101d9383):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1976777852
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1976777852
--- FAIL: TestTenantStreamingMultipleNodes (318.51s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9394665?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9394665?buildTab=artifacts#/) on release-23.1 @ [63b683eab3e0ebfff812bb54fdac936c101d9383](https://github.com/cockroachdb/cockroach/commits/63b683eab3e0ebfff812bb54fdac936c101d9383):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2217391373
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2217391373
--- FAIL: TestTenantStreamingMultipleNodes (313.07s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9419357?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9419357?buildTab=artifacts#/) on release-23.1 @ [35b724947ac464e6af45ce5193e34035fcacb1ef](https://github.com/cockroachdb/cockroach/commits/35b724947ac464e6af45ce5193e34035fcacb1ef):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1350450497
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1350450497
--- FAIL: TestTenantStreamingMultipleNodes (343.32s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9442204?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9442204?buildTab=artifacts#/) on release-23.1 @ [e847e4358c004d4a8977ae26749768ad1d417fba](https://github.com/cockroachdb/cockroach/commits/e847e4358c004d4a8977ae26749768ad1d417fba):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3645054781
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
--- FAIL: TestTenantStreamingMultipleNodes (350.94s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9463270?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9463270?buildTab=artifacts#/) on release-23.1 @ [b432e8c20339de5cfa7c811a9ee6f5dc98d15a1e](https://github.com/cockroachdb/cockroach/commits/b432e8c20339de5cfa7c811a9ee6f5dc98d15a1e):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1142741274
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1142741274
--- FAIL: TestTenantStreamingMultipleNodes (368.82s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9483529?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9483529?buildTab=artifacts#/) on release-23.1 @ [571a1744c72415894dc0e742bd89e74938ea71f4](https://github.com/cockroachdb/cockroach/commits/571a1744c72415894dc0e742bd89e74938ea71f4):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2174277395
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2174277395
--- FAIL: TestTenantStreamingMultipleNodes (336.03s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9508423?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9508423?buildTab=artifacts#/) on release-23.1 @ [de239a7438f44d382c9aefceb65d9c39911dabd2](https://github.com/cockroachdb/cockroach/commits/de239a7438f44d382c9aefceb65d9c39911dabd2):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3270845987
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3270845987
--- FAIL: TestTenantStreamingMultipleNodes (346.80s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9517272?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9517272?buildTab=artifacts#/) on release-23.1 @ [2f96695f75b07c872ec5f146acc1fa198135768f](https://github.com/cockroachdb/cockroach/commits/2f96695f75b07c872ec5f146acc1fa198135768f):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1125718273
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1125718273
--- FAIL: TestTenantStreamingMultipleNodes (345.65s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9555890?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9555890?buildTab=artifacts#/) on release-23.1 @ [c8a4703f9853a442ac676b44d074b43eb387f60c](https://github.com/cockroachdb/cockroach/commits/c8a4703f9853a442ac676b44d074b43eb387f60c):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1704441274
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1704441274
--- FAIL: TestTenantStreamingMultipleNodes (359.40s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9579924?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9579924?buildTab=artifacts#/) on release-23.1 @ [cf84c1c11f2f9c1a4a979033fa0af3befb2ccbcf](https://github.com/cockroachdb/cockroach/commits/cf84c1c11f2f9c1a4a979033fa0af3befb2ccbcf):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2230028440
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2230028440
--- FAIL: TestTenantStreamingMultipleNodes (378.02s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9617784?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9617784?buildTab=artifacts#/) on release-23.1 @ [84905b8396e7cd7ef999b264faaafe13110d631a](https://github.com/cockroachdb/cockroach/commits/84905b8396e7cd7ef999b264faaafe13110d631a):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes591484688
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes591484688
--- FAIL: TestTenantStreamingMultipleNodes (328.37s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9628410?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9628410?buildTab=artifacts#/) on release-23.1 @ [6cce6c746150307a9ecf3b529dfd633a6985c110](https://github.com/cockroachdb/cockroach/commits/6cce6c746150307a9ecf3b529dfd633a6985c110):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3301648202
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3301648202
--- FAIL: TestTenantStreamingMultipleNodes (353.28s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9639807?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9639807?buildTab=artifacts#/) on release-23.1 @ [6cce6c746150307a9ecf3b529dfd633a6985c110](https://github.com/cockroachdb/cockroach/commits/6cce6c746150307a9ecf3b529dfd633a6985c110):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes595926273
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes595926273
--- FAIL: TestTenantStreamingMultipleNodes (352.75s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9660292?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9660292?buildTab=artifacts#/) on release-23.1 @ [12607684868d04f957b3385f142e4e8b9b55c799](https://github.com/cockroachdb/cockroach/commits/12607684868d04f957b3385f142e4e8b9b55c799):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1379670400
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1379670400
--- FAIL: TestTenantStreamingMultipleNodes (342.65s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9681531?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9681531?buildTab=artifacts#/) on release-23.1 @ [2f86475f2e3d713b4c8c976c34dba99acce7c4c7](https://github.com/cockroachdb/cockroach/commits/2f86475f2e3d713b4c8c976c34dba99acce7c4c7):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1861851717
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1861851717
--- FAIL: TestTenantStreamingMultipleNodes (327.26s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9700715?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9700715?buildTab=artifacts#/) on release-23.1 @ [ad6ce866ea3b5c5bb47ba9a0ac19b721a0c98add](https://github.com/cockroachdb/cockroach/commits/ad6ce866ea3b5c5bb47ba9a0ac19b721a0c98add):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3964597765
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3964597765
--- FAIL: TestTenantStreamingMultipleNodes (365.27s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9723363?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9723363?buildTab=artifacts#/) on release-23.1 @ [3919dbf5423e3c9c3313d849580ceb44f4b895fd](https://github.com/cockroachdb/cockroach/commits/3919dbf5423e3c9c3313d849580ceb44f4b895fd):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1979244431
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1979244431
--- FAIL: TestTenantStreamingMultipleNodes (357.58s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9739414?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9739414?buildTab=artifacts#/) on release-23.1 @ [01f129a59ab66166d6ef7189a8930f2b37cced74](https://github.com/cockroachdb/cockroach/commits/01f129a59ab66166d6ef7189a8930f2b37cced74):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2791357424
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2791357424
--- FAIL: TestTenantStreamingMultipleNodes (380.59s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9751724?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9751724?buildTab=artifacts#/) on release-23.1 @ [e82cf4ec488258875dcf5e42e7d6745b03210edd](https://github.com/cockroachdb/cockroach/commits/e82cf4ec488258875dcf5e42e7d6745b03210edd):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2610597982
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2610597982
--- FAIL: TestTenantStreamingMultipleNodes (361.18s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9783110?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9783110?buildTab=artifacts#/) on release-23.1 @ [fa10e980c24442f18e688bea904ece4777ba0ca9](https://github.com/cockroachdb/cockroach/commits/fa10e980c24442f18e688bea904ece4777ba0ca9):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3057357969
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3057357969
--- FAIL: TestTenantStreamingMultipleNodes (352.36s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9827626?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9827626?buildTab=artifacts#/) on release-23.1 @ [6636edf9d8c0ebf9cdeedeeffd563ecb49da5b8a](https://github.com/cockroachdb/cockroach/commits/6636edf9d8c0ebf9cdeedeeffd563ecb49da5b8a):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1527523163
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1527523163
--- FAIL: TestTenantStreamingMultipleNodes (329.72s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9849819?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9849819?buildTab=artifacts#/) on release-23.1 @ [7f6458093d42bb7131cef623deb47f6318a233e9](https://github.com/cockroachdb/cockroach/commits/7f6458093d42bb7131cef623deb47f6318a233e9):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes325102861
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes325102861
--- FAIL: TestTenantStreamingMultipleNodes (326.43s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9869612?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9869612?buildTab=artifacts#/) on release-23.1 @ [f0e37ada00cf99a1fa25236e158861647c2ea713](https://github.com/cockroachdb/cockroach/commits/f0e37ada00cf99a1fa25236e158861647c2ea713):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2858783802
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2858783802
--- FAIL: TestTenantStreamingMultipleNodes (329.38s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9881475?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9881475?buildTab=artifacts#/) on release-23.1 @ [6944c293f14d5c1c5b28d6ea53329c3e214dd9ea](https://github.com/cockroachdb/cockroach/commits/6944c293f14d5c1c5b28d6ea53329c3e214dd9ea):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes96801709
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes96801709
--- FAIL: TestTenantStreamingMultipleNodes (333.51s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9892639?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9892639?buildTab=artifacts#/) on release-23.1 @ [fa8d237846052ae72cc44b3eaebe1f38c89834f8](https://github.com/cockroachdb/cockroach/commits/fa8d237846052ae72cc44b3eaebe1f38c89834f8):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1260487552
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1260487552
--- FAIL: TestTenantStreamingMultipleNodes (348.36s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9905616?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9905616?buildTab=artifacts#/) on release-23.1 @ [82fddc52747a86c696a1f77a85abc25d05858cc1](https://github.com/cockroachdb/cockroach/commits/82fddc52747a86c696a1f77a85abc25d05858cc1):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4223804596
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4223804596
--- FAIL: TestTenantStreamingMultipleNodes (315.43s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9917645?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9917645?buildTab=artifacts#/) on release-23.1 @ [fb4670bf26561963694a5b431549c7ae87bb12cd](https://github.com/cockroachdb/cockroach/commits/fb4670bf26561963694a5b431549c7ae87bb12cd):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4175116578
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4175116578
--- FAIL: TestTenantStreamingMultipleNodes (333.79s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #86206 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [A-disaster-recovery C-test-failure O-robot T-disaster-recovery branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10736407?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10736407?buildTab=artifacts#/) on release-23.1 @ [46b5a2cdc03a1bcbf59aab077cd062a8db62cc1b](https://github.com/cockroachdb/cockroach/commits/46b5a2cdc03a1bcbf59aab077cd062a8db62cc1b):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1318314385
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': dial tcp 127.0.0.1:33621: connect: connection refused
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1318314385
--- FAIL: TestTenantStreamingMultipleNodes (442.64s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10750891?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10750891?buildTab=artifacts#/) on release-23.1 @ [e12e85479312972b551677203849d29aeb38ad5f](https://github.com/cockroachdb/cockroach/commits/e12e85479312972b551677203849d29aeb38ad5f):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes126114287
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': read tcp 127.0.0.1:46074->127.0.0.1:41109: read: connection reset by peer
    panic.go:522: -- test log scope end --
--- FAIL: TestTenantStreamingMultipleNodes (439.07s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10757138?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10757138?buildTab=artifacts#/) on release-23.1 @ [e12e85479312972b551677203849d29aeb38ad5f](https://github.com/cockroachdb/cockroach/commits/e12e85479312972b551677203849d29aeb38ad5f):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes963233390
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': dial tcp 127.0.0.1:46121: connect: connection refused
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes963233390
--- FAIL: TestTenantStreamingMultipleNodes (517.19s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10763796?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10763796?buildTab=artifacts#/) on release-23.1 @ [e12e85479312972b551677203849d29aeb38ad5f](https://github.com/cockroachdb/cockroach/commits/e12e85479312972b551677203849d29aeb38ad5f):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4034190256
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': dial tcp 127.0.0.1:45985: connect: connection refused
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4034190256
--- FAIL: TestTenantStreamingMultipleNodes (417.61s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10771625?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10771625?buildTab=artifacts#/) on release-23.1 @ [a26ee7efd223c92cda531152837302f03a14f461](https://github.com/cockroachdb/cockroach/commits/a26ee7efd223c92cda531152837302f03a14f461):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1267786531
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': dial tcp 127.0.0.1:34205: connect: connection refused
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1267786531
--- FAIL: TestTenantStreamingMultipleNodes (506.15s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10780816?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10780816?buildTab=artifacts#/) on release-23.1 @ [f1108313b2b51ffab176b6b230bcc15b0259f170](https://github.com/cockroachdb/cockroach/commits/f1108313b2b51ffab176b6b230bcc15b0259f170):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3476952680
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': dial tcp 127.0.0.1:40319: connect: connection refused
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3476952680
--- FAIL: TestTenantStreamingMultipleNodes (501.11s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10813870?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10813870?buildTab=artifacts#/) on release-23.1 @ [3eeb4361863fd8ad291a635676b2adf44b6acb4d](https://github.com/cockroachdb/cockroach/commits/3eeb4361863fd8ad291a635676b2adf44b6acb4d):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1277052916
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': dial tcp 127.0.0.1:33899: connect: connection refused
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1277052916
--- FAIL: TestTenantStreamingMultipleNodes (394.29s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10813868?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10813868?buildTab=artifacts#/) on release-23.1 @ [3eeb4361863fd8ad291a635676b2adf44b6acb4d](https://github.com/cockroachdb/cockroach/commits/3eeb4361863fd8ad291a635676b2adf44b6acb4d):

Fatal error:

```
panic: use of Span after Finish. Span: stream-ingestion-processor. Finish previously called at: <stack not captured. Set debugUseAfterFinish>

```
Stack: 

```
goroutine 515827 [running]:
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Span).detectUseAfterFinish(0x73fddb8?)
	github.com/cockroachdb/cockroach/pkg/util/tracing/span.go:182 +0x12e
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Span).Tracer(...)
	github.com/cockroachdb/cockroach/pkg/util/tracing/span.go:225
github.com/cockroachdb/cockroach/pkg/util/tracing.ChildSpan({0x73fddb8, 0xc00e045bc0}, {0x5e54718, 0x16})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1584 +0x46
github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient.(*partitionedStreamSubscription).Subscribe(0xc019105220, {0x73fddb8?, 0xc00e045bc0?})
	github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient/partitioned_stream_client.go:308 +0x65
github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1()
	github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:168 +0x25
golang.org/x/sync/errgroup.(*Group).Go.func1()
	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:75 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:72 +0xa5
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1538837135
    test_log_scope.go:79: use -show-logs to present logs inline
```
</p>
</details>
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10830894?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10830894?buildTab=artifacts#/) on release-23.1 @ [5a53114bf5ae9e5ec672d43acf599f145bf22341](https://github.com/cockroachdb/cockroach/commits/5a53114bf5ae9e5ec672d43acf599f145bf22341):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes943034635
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': dial tcp 127.0.0.1:43957: connect: connection refused
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes943034635
--- FAIL: TestTenantStreamingMultipleNodes (498.44s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10837925?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10837925?buildTab=artifacts#/) on release-23.1 @ [15480b8496e7928845330055d191e331b228eb66](https://github.com/cockroachdb/cockroach/commits/15480b8496e7928845330055d191e331b228eb66):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1845039945
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:346: condition failed to evaluate within 45s: stream ingestion has not recorded any progress yet, waiting to advance pos 1688886776.231145429,0
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1845039945
--- FAIL: TestTenantStreamingMultipleNodes (471.11s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10844493?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10844493?buildTab=artifacts#/) on release-23.1 @ [f62527e83e458f5f4521497e7bfd5c60b70f1e31](https://github.com/cockroachdb/cockroach/commits/f62527e83e458f5f4521497e7bfd5c60b70f1e31):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2466137716
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': dial tcp 127.0.0.1:35607: connect: connection refused
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2466137716
--- FAIL: TestTenantStreamingMultipleNodes (399.25s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10876950?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10876950?buildTab=artifacts#/) on release-23.1 @ [7bfb0ce4cf4e5d3c6c51fd97ced445f4c05fbdc8](https://github.com/cockroachdb/cockroach/commits/7bfb0ce4cf4e5d3c6c51fd97ced445f4c05fbdc8):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1175092655
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:394: error executing 'ALTER TABLE d.scattered SCATTER': dial tcp 127.0.0.1:45055: connect: connection refused
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1175092655
--- FAIL: TestTenantStreamingMultipleNodes (406.60s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10892866?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10892866?buildTab=artifacts#/) on release-23.1 @ [0efd1568d09a5c6a54ae41b0bbf42c682331d24b](https://github.com/cockroachdb/cockroach/commits/0efd1568d09a5c6a54ae41b0bbf42c682331d24b):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1671833762
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:346: condition failed to evaluate within 45s: stream ingestion has not recorded any progress yet, waiting to advance pos 1689231126.745784816,0
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1671833762
--- FAIL: TestTenantStreamingMultipleNodes (578.80s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
+++ b/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
@@ -462,8 +462,8 @@ func TestTenantStreamingDropTenantCancelsStream(t *testing.T) {
 func TestTenantStreamingUnavailableStreamAddress(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.WithIssue(t, 105956)
 
+	skip.UnderDeadlock(t, "multi-node may time out under deadlock")
 	skip.UnderRace(t, "takes too long with multiple nodes")
 
 	ctx := context.Background()
@@ -510,11 +510,6 @@ func TestTenantStreamingUnavailableStreamAddress(t *testing.T) {
 	defer alternateSrcTenantConn.Close()
 	alternateSrcTenantSQL := sqlutils.MakeSQLRunner(alternateSrcTenantConn)
 
-	cleanUpTenant := c.StartDestTenant(ctx)
-	defer func() {
-		require.NoError(t, cleanUpTenant())
-	}()
-
 	alternateCompareResult := func(query string) {
 		sourceData := alternateSrcTenantSQL.QueryStr(c.T, query)
 		destData := c.DestTenantSQL.QueryStr(c.T, query)
@@ -536,6 +531,11 @@ func TestTenantStreamingUnavailableStreamAddress(t *testing.T) {
 	cutoverOutput := replicationtestutils.DecimalTimeToHLC(t, cutoverStr)
 	require.Equal(c.T, cutoverTime, cutoverOutput.GoTime())
 	jobutils.WaitForJobToSucceed(c.T, c.DestSysSQL, jobspb.JobID(ingestionJobID))
+
+	cleanUpTenant := c.StartDestTenant(ctx)
+	defer func() {
+		require.NoError(t, cleanUpTenant())
+	}()
 
 	// The destroyed address should have been removed from the topology
 	progress = jobutils.GetJobProgress(c.T, c.DestSysSQL, jobspb.JobID(ingestionJobID))
@@ -652,6 +652,7 @@ func TestTenantStreamingMultipleNodes(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderDeadlock(t, "multi-node may time out under deadlock")
 	skip.UnderRace(t, "takes too long with multiple nodes")
 
 	ctx := context.Background()

